README.md

```diff
@@ -21,11 +21,32 @@ This project makes running the InstructLab large language model (LLM) fine-tunin
       * NOTE: v3 is not compatible with RHOAI
 * [SDG taxonomy tree] to utilize for Synthetic Data Generation (SDG)
 * An OpenShift AI 2.19 or newer installation, with:
-  * Training Operator, ModelRegistry, KServe, and Data Science Pipelines components installed via the DataScienceCluster
+  * Training Operator, Model Registry, KServe, and Data Science Pipelines components installed via the DataScienceCluster
     * See docs on [Installing RHOAI components via DSC]
   * For [Model Registry] you will need:
     * Model Registry API URL
     * Model Registry Name
+    * The `pipeline-runner-dspa` `ServiceAccount` must be assigned the `registry-user-<model registry name>` role in
```

Comment:
This whole section could be replaced by something like:

_Configure Model Registry permissions to provide all service accounts in your data science project access to the model registry, following the instructions at [Managing model registry permissions](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.20/html/managing_model_registries/managing-model-registry-permissions_managing-model-registries)_

Reply:
I personally prefer documenting through Kubernetes manifests rather than the UI.

Reply:
Done